### PR TITLE
X2-6961: Scroll bar to log out from x2-seller is not visible properly

### PR DESCRIPTION
**Pull request by VitalyDevico**

### CODE REVIEW

File: src/components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -64,17 +75,26 @@ export const Sidebar = ({ logo, children, className, footer, notifications, isFi
...
+    const isStickyHeaderFooter = isStickyHeader && isStickyFooter;
+
     return (
         <div
             className={clsx(
+                sidebarScroll,
                 "ui-sidebar",
                 isFixed ? "fixed" : "relative",
-                "z-20 flex h-full w-16 flex-col overflow-y-auto bg-black py-2 px-1 text-white md:w-24 xl:w-50",
+                !isStickyHeaderFooter && "overflow-y-auto",
```
Comment (KarthickXola) [inline src/components/Sidebar/Sidebar.jsx:86]: py-2 px-1 why this not needed now

File: src/components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -12,13 +12,24 @@ import { SidebarFooter } from "./Sidebar.Footer";
...
-export const Sidebar = ({ logo, children, className, footer, notifications, isFixed = true, onLogoClick }) => {
+export const Sidebar = ({
+    logo,
+    children,
+    className,
+    footer,
+    notifications,
+    isFixed = true,
+    onLogoClick,
+    isStickyHeader = true,
```
Comment (KarthickXola) [inline src/components/Sidebar/Sidebar.jsx:30]: not sure, did we need to have Sticky header by default  ?

**Review by KarthickXola — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/Sidebar.jsx (hunk 2 above)
Comment (VitalyDevico) [reply src/components/Sidebar/Sidebar.jsx:30]: In The ticket mentioned - _the flexible part of the left menu should only be below the settings tab and above the logo_
so everything above the logo is sticky

File: src/components/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (VitalyDevico) [reply src/components/Sidebar/Sidebar.jsx:86]: this created a gap under the footer and above the header which resulted in an issue when menu items were visible after scrolling top or bottom

**Review by KarthickXola — APPROVED** [review]

**Pull request merged**